server/server.js

```diff
@@ -20,9 +20,29 @@ server.listen(config.port);
 log.info('Server is on bruh and running on port: '+ config.port);
 
 //----------------------------------
+// Allow Cors Domain:
```

Comment:
In fact a good article about documentation that should probably be read is this one; https://dev.to/andreasklinger/comments-explain-why-not-what-and-2-more-rules-on-writing-good-comments